### PR TITLE
Setup docker

### PR DESCRIPTION
**Pull request by seiferla**
I#ve created a Docker file for our mail server
#2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RUN mvn -f /home/app/pom.xml clean package
+RUN ls /home/app/target/
+
+#
+# Package stage
+#
+FROM eclipse-temurin:17
+COPY --from=build /home/app/target/mailflow-1.0-SNAPSHOT.jar /usr/local/lib/mailflow.jar
+EXPOSE 587
+EXPOSE 465
```
Comment (Jonas-Karl) [inline Dockerfile:16]: Is this necessary?

File: docs/MailFlow_Klassendiagramm.drawio
Comment (Jonas-Karl) [inline docs/MailFlow_Klassendiagramm.drawio:1]: This should not be part of this PR

File: docs/MailFlow_Klassendiagramm.png
Comment (Jonas-Karl) [inline docs/MailFlow_Klassendiagramm.png:1]: This should also not be part of this PR

**Review by Jonas-Karl — COMMENTED** [review]
I did not test it yet but it looks good besides of the diagrams

File: Dockerfile (hunk 1 above)
Comment (seiferla) [reply Dockerfile:16]: Yes, these are the ports of smtp

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#
+# Build stage
+#
+FROM maven:3.9.5-eclipse-temurin-17-focal AS build
+COPY src /home/app/src
+COPY pom.xml /home/app
+RUN mvn -f /home/app/pom.xml clean package
+RUN ls /home/app/target/
```
Comment (jens1o) [inline Dockerfile:8]: unnecessary line that can be removed :)

**Review by jens1o — APPROVED** [review]
apart from the unnecessary, but not necessarily hurting, line, lgtm! Thanks for your work!

File: Dockerfile (hunk 2 above)
Comment (seiferla) [reply Dockerfile:8]: I'm confused. You marked 4 lines and wrote "Unnecessary line". 
Should I remove one line or the marked lines?
Sorry for this stupid question

Comment (jens1o) [reply Dockerfile:8]: Oh. Sorry, that's by mistake.

I only meant `RUN ls /home/app/target/`. The other ones _are_ necessary. :D
